kube-config-files/oneshot/neo4j-red-restore-job.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: neo4j-red-restore
+spec:
+  template:
+    metadata:
+      name: neo4j-red-restore
+    spec:
+      containers:
+      - name: neo4j-red-restore
+        image: coco/neo4j-hot-backup
+        command: 
+          - ./neo4j-hot-backup
+          - restore 
+          - 2017-02-09T14-00-33
```

Comment:
The neo4j is going to be moved out of the cluster, so this is only a temporary solution. We would have to either change this file, of change a property on the cluster anyway to configure this. It's not ideal but this won't be used in a prod setting.